src/utils/narrow.js

```diff
@@ -182,6 +182,12 @@ export const isGroupNarrow = (narrow?: Narrow): boolean =>
 export const emailsOfGroupNarrow = (narrow: Narrow): string[] =>
   caseNarrowPartial(narrow, { groupPm: emails => emails });
 
+/**
+ * The recipients' email if a Private narrow; else error.
```

Comment:
Nit: "recipient's" (apostrophe before the "s"); if I understand correctly, this is one person's email.